### ISSUE TITLE
invalid literal for int() with base 10: 'c'

### ISSUE THREAD

**Issue by milesabarr** [switch-model/switch#68]
Hi all,

I am interested in using your SWITCH tool with research on alternative ES and generation systems. I tried setting up SWITCH, but am running into the following errors when I run the run_tests.py:

```
invalid literal for int() with base 10: 'c'
invalid literal for int() with base 10: 'c'
invalid literal for int() with base 10: 'c'
invalid literal for int() with base 10: 'c'
invalid literal for int() with base 10: 'c'
invalid literal for int() with base 10: 'c'
invalid literal for int() with base 10: 'c'
invalid literal for int() with base 10: 'c'
invalid literal for int() with base 10: 'c'
invalid literal for int() with base 10: 'c'
invalid literal for int() with base 10: 'c'
invalid literal for int() with base 10: 'c'
invalid literal for int() with base 10: 'c'
To exit: use 'exit', 'quit', or Ctrl-D.
An exception has occurred, use %tb to see the full traceback.

SystemExit: True
```

I will admit, trying to get SWITCH set up was a bit confusing for me, so this could very well be an issue on my end. To help debug this issue, here are some details about what I have installed and the process I took to set up SWITCH:

Environment:
- Windows 10
- Python 2.7.12 :: Anaconda 4.1.1 (32-bit)
- Spyder IDE

Process to set up SWITCH:
- Long ago I had tried installing Pyomo, so that was already on my computer. I also had gone through the process to install the GLPK solver. I remember that being a huge headache, but I believe I did get that fully installed based on glpsol --help working from any directory in the Windows command prompt (see http://www.pyomo.org/blog/2015/8/24/checking-glpk-installation). I wish I could remember the exact process I used, but I don't recall. When I type glpsol --version in the command prompt it says: GLPSOL: GLPK LP/MIP Solver, v4.59
- I forked switch-model/switch to my own GitHub
- In the Windows command prompt within the switch folder I used: pip install --user -r pip_requirements.txt
- In the Windows command prompt I used: python setup.py develop
- Added all switch folders/subfolders to my PYTHONPATH
- I have tried going to https://projects.coin-or.org/Cbc multiple times, but it keeps timing out

After all of this, I try running run_tests.py in Spyder and it gives me the above errors. Any ideas what the problem could be? Thanks for any help.

Miles

Edit: I forgot to also mention that I have not been able to successfully run make_doc.sh. I downloaded cygwin and from there went to the doc folder and tried 

`bash make_doc.sh`

It then gives me:

`no Python documentation found for '../\r'`

**Comment by bmaluenda** [comment]
Hello Miles,

There was an issue with Pyomo 4.3x and below with GLPK newest versions. That bug was apparently only fixed in version 4.41 [(check this stackoverflow answer)](http://stackoverflow.com/a/39111341/6067818), so I would recommend upgrading Pyomo. This can be done with the Pip package manager: `pip install -U pyomo`

The error the `make_doc.sh` script is raising probably has something to do with the Unix bash shell trying to interpret a line ending character that Windows read from the script. Line endings in Windows consist on the characters `\r\n`, while Unix only uses `\n`. Cygwin is probably not parsing line endings correctly. A quick solution so you can generate the documentation can be to directly execute the command `pydoc -w ../` in the Cygwin shell.

Documentation written out by Pydoc is not really that friendly, so I would recommend skimming through the scripts as well. We are aiming to have better documentation some time soon.

You can write out any other general questions to the [Switch Model GoogleGroup](https://groups.google.com/forum/?hl=es#!forum/switch-model) to get access to everyone else who is working with the package. New voices are appreciated.

**Comment by milesabarr** [comment]
Benjamin,

Thank you for the assistance. I finally got around to finding time to work on this more. I did what you said, and it seems things are working. When I run run_tests.py I get:

```
runfile('C:/Users/miles/OneDrive/Documents/GitHub/switch/run_tests.py', wdir='C:/Users/miles/OneDrive/Documents/GitHub/switch')
To exit: use 'exit', 'quit', or Ctrl-D.
An exception has occurred, use %tb to see the full traceback.

SystemExit: True
```

I am assuming this is desired? The reason I am assuming things are fine is I ran the copperplate0 example in the command prompt and it solves the problem and gives me outputs. I have other questions on how to view the results, and if you always have to run the problems from the command prompt, but I will investigate more myself first and see what I can figure out. I am sure I will have more questions, but I will post those in the Google Group if they are general like this. Thanks again! I am very excited to start working with your SWITCH model.

Miles
